dev/automated_tests/flutter_test/ticker_test.dart

```diff
@@ -7,10 +7,14 @@ import 'package:flutter/services.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 void main() {
+  setUp(() {
```

Comment:
Why do we need to manually call this here? Isn't the WidgetTester already calling this for us?

https://github.com/flutter/flutter/blob/b11c5acaa6200a8d662705a312a2e5fb7714fe2e/packages/flutter_test/lib/src/widget_tester.dart#L105

Reply:
(Same for some of the other updated tests below)

Reply:
You are right. Reverted some of the files. We'll need to explicitly call `TestWidgetsFlutterBinding.ensureInitialized();` when:
- It's a Dart unit tests using `test()`;
- System channels are used before WidgetTester is initialized.